lib/tty/logger/handlers/console.rb

```diff
@@ -72,13 +72,14 @@ class Console
         attr_reader :level
 
         def initialize(output: $stderr, formatter: nil, config: nil, level: nil,
-                       styles: {})
+                       styles: {}, message_padding: 25)
```

Comment:
feel free to suggest a better place to inject this

Reply:
Please see my later comment which suggests using `message_format` as a configuration option.